### PR TITLE
Update hardware requirements

### PR DESCRIPTION
**Pull request by janewang**

### CODE REVIEW

**Comment by janewang** [comment]
@chefsale  What's the correct machine size for AWS, GCP and Azure? Thanks

File: docs/develop/node/archival/hardware-archival.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ Estimated monthly costs depending on operating system:
 
 | Cloud Provider | Machine Size    | Linux                     |
 | -------------- | --------------- | ------------------------  |
-| AWS            | c5.2xlarge      | $250 CPU + $300 storage † |
-| GCP            | c2-standard-8   | $220 CPU + $300 storage † |
-| Azure          | Standard_F8s_v2 | $180 CPU + $300 storage † |
+| AWS            | c5.2xlarge      | $350 CPU + $300 storage † |
+| GCP            | e2-standard-8   | $320 CPU + $300 storage † |
```
Comment (nikurt) [inline docs/develop/node/archival/hardware-archival.md:39]: Let's keep `c2-standard-8`, which a instance family for compute-intensive applications.
This makes it more consistent with Azure and AWS instance types.

File: docs/develop/node/rpc/hardware-rpc.md [hunk 2]
```diff
@@ -36,9 +36,9 @@ Estimated monthly costs depending on operating system:
 
 | Cloud Provider | Machine Size    | Linux                  |
 | -------------- | --------------- | ---------------------- |
-| AWS            | c5.2xlarge      | $250 CPU + $20 storage |
-| GCP            | c2-standard-8   | $220 CPU + $20 storage |
-| Azure          | Standard_F8s_v2 | $180 CPU + $10 storage |
+| AWS            | c5.2xlarge      | $350 CPU + $20 storage |
+| GCP            | e2-standard-8   | $320 CPU + $20 storage |
```
Comment (nikurt) [inline docs/develop/node/rpc/hardware-rpc.md:40]: Let's keep `c2-standard-8`, which a instance family for compute-intensive applications.
This makes it more consistent with Azure and AWS instance types.

File: docs/develop/node/archival/hardware-archival.md (hunk 1 above)
Comment (janewang) [reply docs/develop/node/archival/hardware-archival.md]: Done.

File: docs/develop/node/rpc/hardware-rpc.md (hunk 2 above)
Comment (janewang) [reply docs/develop/node/rpc/hardware-rpc.md]: Done.

**Review by nikurt — APPROVED** [review]

**Comment by bucanero** [comment]
Just for the record, the Wiki article might also need an update: https://wiki.near.org/validator/validator-docs/hardware-requirements

Source: https://github.com/near/wiki/blob/master/validator/validator-docs/hardware-requirements.md

**Comment by janewang** [comment]
> Just for the record, the Wiki article might also need an update: https://wiki.near.org/validator/validator-docs/hardware-requirements
> 
> Source: https://github.com/near/wiki/blob/master/validator/validator-docs/hardware-requirements.md

Who's the owner of the Wiki? @bucanero

**Comment by bowenwang1996** [comment]
Thanks for updating this @janewang !

**Comment by bucanero** [comment]
> > Just for the record, the Wiki article might also need an update: https://wiki.near.org/validator/validator-docs/hardware-requirements
> > Source: https://github.com/near/wiki/blob/master/validator/validator-docs/hardware-requirements.md
> 
> Who's the owner of the Wiki? @bucanero

Hi @janewang, 

Illia has been handling the Wiki project (@ilblackdragon). You can check the details here: https://gov.near.org/t/done-establishing-near-wiki/3409

You can request editor access to GitBook, or submit PRs to the wiki's Github repo. Once the changes are merged, they get published on wiki.near.org.

I have updated the Wiki with the new hardware requirements to speed up things and save some time. 😄

**Pull request merged**